substratee-node-primitives/src/lib.rs

```diff
@@ -23,6 +23,11 @@ pub type SubstrateeConfirmCallFn = ([u8; 2], ShardIdentifier, H256, Vec<u8>);
 pub type ShieldFundsFn = ([u8; 2], Vec<u8>, u128, ShardIdentifier);
 pub type CallWorkerFn = ([u8; 2], Request);
 
+// Polkadex Types
+pub type OCEXRegisterFn = ([u8; 2], [u8; 32]);
```

Comment:
AccountId32? Or rather the AccoundId defined in polkadex primitives? In case the Account type will change, due to whatever reason, it would be helpful to have only one type that needs to be redefined and not every function.

Reply:
I am using AccountId from my_node_runtime

Reply:
We will be fixing this in a separate PR